scripts/set_deployment_image_tag.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Generic script that sets the image+tag of a container in a deployment's pod spec in user namespaces
+# As is.  Will set deployment's "r-studio-server" container's image tag to "v1.3.2" in all user namespaces
+
+set -e
+
+DRY_RUN=true
+
+while true; do
+	read -p "Is this a DRY RUN? i.e. NO-OP [y/N]: " dry_run
+	case $dry_run in
+	    [y]*    ) DRY_RUN=true; break;;
+	    [N]*    ) DRY_RUN=false; break;;
+	    *       ) echo "Answer y to simulate the operation.  Answer N to perform the operation."; exit;;
+    esac
+done
+
+APP=rstudio
+TAG=v1.3.2
```

Comment:
I saw this in the helm-charts repo as well, I think you were right about having this in the `*-ops` repo until we find a better place. 

I would also add a param for the `$IMAGE`, as I've [done here](https://github.com/ministryofjustice/analytics-platform-helm-charts/commit/596ba662098914b55a0439dae91f2b7553f14827#diff-fbf8fa4423f8d08f8e559443cea7f5df), so we can potentially update the other container image (e.g. the proxy) and probably delete the script in the helm-charts repo as it's a duplication now :)